api/.prettierrc

```diff
@@ -2,14 +2,7 @@
   "semi": false,
   "printWidth": 130,
   "singleQuote": true,
+  "useTabs": true,
```

Comment:
Again, I don't think we should be using tabs as we're currently using spaces and this will affect every single file in the codebase.

Reply:
Again, definitely agree with @paul-graham on this one.